bin/vaultclient

```diff
@@ -213,20 +213,25 @@ program
 
 program
     .command('update-account-quota')
-    .option('--account-name <NAME>', 'Name of the account')
     .option('--quota <QUOTA>', 'Maximum quota for the account', parseInt)
+    .option('--account-name <NAME>', 'Name of the account')
```

Comment:
why this change?

Reply:
this was added to enable the requester to give an account name it is not mandatory rather optional , cf design part : 
![image](https://github.com/scality/vaultclient/assets/144012792/813f7d94-cddc-4559-bc3f-ce1cbc1b9ce4)